config/webpack/webpack.dev.ts

```diff
@@ -1,18 +1,26 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import type webpack from 'webpack';
+import getCommonConfig from './webpack.common';
+
+import portfinder = require('portfinder');
+import TimeAnalyticsWebpackPlugin = require('time-analytics-webpack-plugin');
+
+const {TimeAnalyticsPlugin} = TimeAnalyticsWebpackPlugin;
 const path = require('path');
-const portfinder = require('portfinder');
 const {DefinePlugin} = require('webpack');
 const {merge} = require('webpack-merge');
```

Comment:
cant we change those imports to es modules?

Reply:
Done!